src/main/java/lach_01298/qmd/accelerator/LinearAcceleratorLogic.java

```diff
@@ -298,11 +299,121 @@ public boolean isMachineWhole(Multiblock multiblock)
 
 		return postions;
 	}
-	
-	
-	
+
+	public void updateNumbers()
+	{
+		getAccelerator().RFCavityNumber = getAccelerator().getRFCavityMap().size();
+		getAccelerator().quadrupoleNumber = getAccelerator().getQuadrupoleMap().size();
+		getAccelerator().dipoleNumber = getAccelerator().getDipoleMap().size();
+	}
+
 	// Multiblock Methods
-	
+
+
+
+	public void afterManageRfCavity()

```

Comment:
What's all this for?

Reply:
Because of resetting stats after rf cavity disable/enable, i have to run this code in order to have correct data
But i think, i should not reset everything, just voltage